proto/tendermint/farming/v1beta1/farming.proto

```diff
@@ -157,15 +157,9 @@ message Reward {
   option (gogoproto.goproto_getters)  = false;
   option (gogoproto.goproto_stringer) = false;
 
-  // farmer defines the bech32-encoded address of the staker for the plan
-  string farmer = 1;
-
-  // staking_coin_denom is denom of staked coin as a source of the reward
-  string staking_coin_denom = 2 [(gogoproto.moretags) = "yaml:\"staking_coin_denom\""];
-
```

Comment:
We need to original `Reward` struct for export/import genesis  
Now setting original `Reward` itself on kv, but I'll do research later and add it when saving `RewardCoin` only.